Gruntfile.coffee

```diff
@@ -13,6 +13,9 @@ module.exports = (grunt) ->
   grunt.initConfig
     bump:
       options:
+        commitFiles: ['-a']
+        files: ['package.json']
+        prereleaseName: 'alpha'
```

Comment:
This is totally fine and not a big deal, but I have been thinking about switching everything to use `prereleaseName: 'dev'`. Thoughts?